### ISSUE TITLE
Mesh material ID should be UINT64_MAX for no material

### ISSUE THREAD

**Issue by re-cheid** [roboticeyes/openrex#34]
Specification states:
> If the materialId is INT64_MAX (Long.MAX_VALUE in Java), then no material is available.

the material ID actually is a `uint_64`

**Comment by breiting** [comment]
`#define REX_NOT_SET 0x7fffffffffffffffL`

this is the value which is used in the openrex c library. Maybe we should change the specification to that value. @re-cheid which value is mobile  checking, i hope the same.

**Comment by re-cheid** [comment]
yes, but that's a value in the middle of the `uint_64` range which is a bit odd ;)

**Comment by breiting** [comment]
right, but this is how all .rex files are currently encoded. So I would stick to that value.

**Comment by breiting** [comment]
@re-polm as discussed, please change the documentation accordingly (explicitly use the value above)
